substrate/cmd/kitctl/bootstrap.go

```diff
@@ -15,15 +15,14 @@ limitations under the License.
 package main
 
 import (
-	"os"
-	"time"
-
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/awslabs/kubernetes-iteration-toolkit/substrate/pkg/apis/v1alpha1"
 	"github.com/awslabs/kubernetes-iteration-toolkit/substrate/pkg/controller/substrate"
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/pkg/logging"
+	"os"
```

Comment:
Looks like the IDE settings are different which are moving these imports from the std library.